src/go/k8s/cmd/configurator/main.go

```diff
@@ -161,6 +173,23 @@ func main() {
 
 var errInternalPortMissing = errors.New("port configration is missing internal port")
 
+func getZoneLabels(nodeName string) (zone, zoneID string, err error) {
+	node, err := getNode(nodeName)
+	if err != nil {
+		return "", "", fmt.Errorf("unable to retrieve node: %w", err)
+	}
+	zone = node.Labels["topology.kubernetes.io/zone"]
+	zoneID = node.Labels["topology.cloud.redpanda.com/zone-id"]
```

Comment:
suggestion: I saw there is a daemonset that sets this, but maybe safer to check if this exists to avoid panic

Reply:
It won't panic though. If the key is missing, the map would return an empty string.